pkgs/servers/misc/qremotecontrol-server/default.nix

```diff
@@ -1,21 +1,35 @@
-{ lib, stdenv
-, fetchurl
-, qmake4Hook
-, qt4
+{ lib
+, stdenv
+, fetchgit
+, qmake
+, wrapQtAppsHook
+, qtbase
 , xorg
 }:
 
 stdenv.mkDerivation rec {
   pname = "qremotecontrol-server";
-  version = "2.4.1";
+  version = "unstable-2014-11-05"; # basically 2.4.2 + qt5
 
-  src = fetchurl {
-    url = "mirror://sourceforge/project/qrc/${version}/qremotecontrol-${version}.tar.bz2";
-    sha256 = "07hzc9959a56b49jgmcv8ry8b9sppklvqs9kns3qjj3v9d22nbrp";
+  src = fetchgit {
+    url = "https://git.code.sf.net/p/qrc/gitcode";
+    rev = "8f1c55eac10ac8af974c3c20157d90ef57f7308a";
+    sha256 = "sha256-AfFScec5/emG/f+yc5Zn37USIEWzGP/sBifE6Kx8d0E=";
```

Comment:
I get
```
error: hash mismatch in fixed-output derivation '/nix/store/0c2dp4khkq58hpy350f4q2qqiaqmpznb-gitcode-8f1c55e.drv':
         specified: sha256-AfFScec5/emG/f+yc5Zn37USIEWzGP/sBifE6Kx8d0E=
            got:    sha256-fdUiuW5mdXWewvrJ2LKuEnB2qrPPALSg/cjOEBnc5YY=
```
:thinking:

Reply:
weird… suggests that the hash isn't stable, which it really shouldn't be.

Reply:
I'm still getting the same hash...

Reply:
I rebased. That might do the trick, like in #179560